main.go

```diff
@@ -53,6 +53,7 @@ import (
 	"github.com/awalterschulze/goderive/plugin/uncurry"
 	"github.com/awalterschulze/goderive/plugin/union"
 	"github.com/awalterschulze/goderive/plugin/unique"
+	"github.com/ingun37/goderive/plugin/toerror"
```

Comment:
Yeah this is weird problem with Go.
What I do in this case is:
```
cd $GOPATH/src/github.com/awalterschulze
git clone https://github.com/ingun37/goderive 
```
Then your import can be 
```
github.com/ingun37/goderive/plugin/toerror
```